app/models/user_profile.rb

```diff
@@ -75,6 +76,36 @@ def rebake!
     update_columns(bio_cooked: cooked, bio_cooked_version: BAKED_VERSION)
   end
 
+  def self.import_url_for_user(background_url, user, options = nil)
```

Comment:
Test case for this method please? :)

For reference: https://github.com/discourse/discourse/blob/ee69d58a59c824a7a0b1146710930c5861d3ca26/spec/models/user_avatar_spec.rb#L19-L62